pkg/roachpb/metadata_replicas_test.go

```diff
@@ -189,7 +189,13 @@ func TestReplicaDescriptorsConfState(t *testing.T) {
 		t.Run("", func(t *testing.T) {
 			r := MakeReplicaSet(test.in)
 			cs := r.ConfState()
-			require.Equal(t, test.out, raft.DescribeConfState(cs))
+			describeConfState := func(state raftpb.ConfState) string {
+				return fmt.Sprintf(
```

Comment:
nit: factor it out as func, or use the original (moved to another package so that circular dep is resolved)